index.js

```diff
@@ -52,6 +56,18 @@ function transact (fn, cb) {
   })
 }
 
+function extractRequestClient (req, transact) {
+  if (transact.length) {
```

Comment:
invert the condition here to return early, plus use a more obvious condition: `typeof transact === 'string'`